README.md

```diff
@@ -22,15 +22,19 @@ conda install -c numba -c conda-forge -c rapidsai -c defaults cudf=0.2.0
 
 Note: This conda installation only applies to Linux and Python versions 3.5/3.6.
 
-You can create and activate a development environment using the conda command:
+You can create and activate a development environment using the conda commands:
 
 ```bash
-conda env create --name cudf --file conda_environments/testing_py35.yml
-source activate cudf
+# create the conda environment (assuming in base `cudf` directory)
+$ conda env create --name cudf_dev --file conda_environments/dev_py35.yml
```

Comment:
This filepath to the yml is incorrect. I had to do:

```conda env create --name cudf_dev --file conda/environments/dev_py35.yml```

Reply:
Resolved.